README.rst

```diff
@@ -11,22 +11,47 @@ lcls_naming_tool
 
 `Documentation <https://pcdshub.github.io/lcls_naming_tool/>`_
 
-A tool that checks the form and content of names with respect to the LCLS naming convention.
+A tool that checks the form and content of PV and device names with respect to the LCLS naming convention.
+
+Instructions
+------------
+
+Use at the command line:
+
+1. Activate the custom virtual environment with ``source /cds/group/pcds/pyps/conda/venvs/lcls_naming_tool/bin/activate``
+
+2. To check if a PV or device name is valid pipe in the name like so ``$ echo "MR2K4:KBO:PIP:01:PUMPSIZE" | ./lcls_naming_tool.py``
+
+3. To check if a list of names is valid pipe in the file name like so ``$ cat pvlist.txt | ./lcls_naming_tool.py`` (names should be separated by newline characters.)
+
+4. To view the current version add ``-v`` or ``--version`` like so ``$ echo "MR2K4:KBO:PIP:01:PUMPSIZE" | ./lcls_naming_tool.py -v``
+
+To run Flask and Gunicorn web server:
+
+``ssh psca@psctlws01``
+
+``cd /u1/psca/test/apps/nmsvc``
+
+``git pull``
+
+``./startup.sh``
+
+Website is hosted at https://psdm.slac.stanford.edu/lcls_naming_tool/
+
 
 Requirements
 ------------
 
+* PCDS conda
 * Python 3.9+
+* Flask 2.2.5 (web only)
+* Gunicorn 23.0.0 (webs only)
+
 
 Installation
 ------------
```

Comment:
I think its OK for these sections to be empty for now, but we'll need to build some python installer stuff to get this into the conda environment later.

Reply:
Made a note for the next PR.